.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: generic
-sudo: required
-dist: trusty
+dist: xenial
```

Comment:
It's a newer faster image, I figured I'd update it while I was here since we no longer need `sudo required` which forced our hand on using the older "trusty" image.